modules/file-rotation/file-rotation.c

```diff
@@ -22,29 +22,46 @@
  */
 
 #include "file-rotation.h"
+#include "modules/http/http-signals.h"
+#include "modules/affile/file-signals.h"
 #include "driver.h"
 
-struct _FileRotationPlugin
+static void
+_slot_append_file_rotation_request(FileRotationPlugin *self, FileFlushSignalData *data)
```

Comment:
I'd call this function `_slot_file_rotation` or anything similar.

(We can't talk about "appending requests" here, those were just examples of the HTTP signaling).

Reply:
Updated :+1: